src/components/common/ErrorComponent.tsx

```diff
@@ -148,24 +164,29 @@ const ErrorComponent: React.FC<PropsFromRedux> = ({
   return errorType ? (
     <View style={styles.container}>
       <View style={styles.row}>
-        <Text style={styles.title}>{messages[errorType].title}</Text>
+        <Text style={styles.title}>{messages[errorType]?.title}</Text>
         <AccessibleTouchableOpacity
           onPress={() => setErrorType(undefined)}
           style={styles.closeButton}>
           <Icon name="close-outline" size={24} color={BLACK} />
         </AccessibleTouchableOpacity>
       </View>
-      {errorType === 'noInternet' ? (
-        <Text style={styles.text}>{messages[errorType].additionalInfo}</Text>
+      {['noInternet', 'outOfServiceArea'].includes(errorType) &&
+      messages[errorType]?.additionalInfo ? (
+        <Text style={styles.text}>{messages[errorType]?.additionalInfo}</Text>
       ) : (
-        <AccessibleTouchableOpacity
-          onPress={() => tryAgainFunctions[errorType]()}>
-          <View style={styles.tryAgainButton}>
-            <View style={styles.textContainer}>
-              <Text style={styles.buttonText}>{t('tryAgain')}</Text>
-            </View>
-          </View>
-        </AccessibleTouchableOpacity>
+        <>
+          {errorType !== 'outOfServiceArea' && errorType !== 'noInternet' && (
```

Comment:
These conditions do not seem necessary but TypeScript complains about `tryAgainFunctions[errorType]` on line 181 if these are omitted.